packages/block-library/src/gallery/edit.js

```diff
@@ -622,9 +670,12 @@ function GalleryEdit( props ) {
 					<ToolbarDropdownMenu
 						icon={ linkIcon }
 						label={ __( 'Link' ) }
+						toggleProps={ {
+							isPressed: hasLinkTo,
+						} }
```

Comment:
For now, I don't think the pressed state is necessary. Even if the link settings are changed in the Gallery block, the Image blocks inside may not follow those settings.

Reply:
This is to indicate the `LinkTo` option is set or not.

Reply:
Even if the Gallery block has the Link To option set, the Image blocks inside may not link to anything. In this case, I'm concerned that users may be confused if we highlight the `Link To` option in the pressed state.

What do you think?

Reply:
You are right! This is little confusing to me too.